sql/codegen.go

```diff
@@ -211,56 +211,53 @@ func newFiller(pr *extendedSelect, ds *trainAndValDataset, fts fieldTypes, db *D
 		}
 	}
 
-	return fillDatabaseInfo(r, db)
+	r.connectionConfig, err = newConnectionConfig(db)
+	if err == nil && r.Driver == "hive" {
+		// remove the last ';' which leads to a (hive)ParseException
+		r.TrainingDataset = strings.TrimSuffix(r.TrainingDataset, ";")
```

Comment:
Maybe rename `TrainingDataset` to `TrainingDatasetSQL` is more meaningful

Reply:
Good idea.